### DISCUSSION TITLE
随机图无法正常显示的问题，以及图床没有删除已上传图片的功能

### DISCUSSION THREAD

**Discussion by Lentinel** [LIlGG/halo-theme-sakura#600] [Announcements]
### 目前使用的 Sakura 版本？

2.4.1

### 目前使用的 halo 版本？

 2.20.10

### 建议/问题

首先我在图床里上传了我的相册和图片：
![image](https://github.com/user-attachments/assets/f94a0a1e-e7c8-418d-a688-b6128448dd89)
相册放置一天后，在设置里填写了相册的编号：
![image](https://github.com/user-attachments/assets/0757aa4f-f4f2-4b22-81f6-a833d0cdf33b)
首页背景图无法正常显示：
![image](https://github.com/user-attachments/assets/d0be6489-7251-4980-8143-09e3225c3d63)
页面背景图问题同上，只有将图库获取范围改为“分类”时才能正常显示**部分**图片：
![image](https://github.com/user-attachments/assets/38e107ba-6b01-4610-916f-70aa6a8975e4)
这个问题已经很久了。（其实我觉得像wordpress版的sakura主题，内带了一个返回随机图片的 API，把想要展示的图放到 /cover/gallery 中即可，这样的功能比图床要更节省资源，自由度也更高。）
另外，https://view.lixingyong.com  大佬的图床似乎没有删除图片的功能？

**Comment by LIlGG** [comment]
你的相册 ID 填写错了。

<img width="639" alt="image" src="https://github.com/user-attachments/assets/04590801-c219-4d4c-87f8-5fd1598fbb8e">

正常的 ID 应当为 `JVpl`

实际上后续我有计划考虑编写插件，来将用户自己网站图床中的图片作为随机图返回，而不是使用我的托管服务，这样对于我个人的服务器网络压力也会小很多。

> 另外，[view.lixingyong.com](https://view.lixingyong.com/) 大佬的图床似乎没有删除图片的功能？

没有允许删除图片的原因是，防止有人上传违规图片传播后删除。如果想删除图片，可以随时联系我进行删除。
